guides/V1_AppActionCallback.md

```diff
@@ -0,0 +1,39 @@
+---
+copyright: 'Copyright IBM Corp. 2017'
+link: 'client-action-handler-callback'
+is: 'experimental'
+---
+
+# Client Action Handler Callback API
+Client action handlers allow your App to deliver a user experience to a Watson Work Services client. You will need to 
+register a call back URL that the client can invoke for specified actions so that the client can recognize and enable 
+your action for users. This callback API needs to be implemented for your app to process the client action. Once a user 
+invokes your action through the client, your App can access context. Learn more by reading 
+[Actions and Events](../guides/V1_ActionsEvents.md).
+
+## URI scheme
+_Host_ : **api.watsonwork.ibm.com**
+_Scheme_: **HTTPS**
+
+## Path
+```
+actionURL?actionHandlerContextToken=<token>
+```
+
+The **actionURL** is the one that was registered for the app. It is required to be a fully qualified URL using the 
+**https** protocol.
+
+The **actionHandlerContextToken** url parameter contains the client action handler context token which has to be used 
+to get the action context information via Watson Work Services, see 
+[Get action context information](./V1_GetActionContext.md) and also 
+[Actions and Events](../guides/V1_ActionsEvents.md). This action handler token is generated by Watson Work 
+Services when a Watson Work Services client attempts to invoke a registered client action handler that has been 
+registered by your app.
+
+
+
+### Response
+
+HTTP status code **200 OK**
```

Comment:
Do they need to send this 200 back?  What happens if they don't send 200?

Reply:
because this callback url will be opened in browser tab, so we do not care about the response status.
I'm ok that we delete the status code here, I just follow the format of existing app configuration doc https://developer.watsonwork.ibm.com/docs/apps/app-configuration-callback